pyrightconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ignore": ["./starknet_in_rust/", "./blockifier/"]
+}
```

Comment:
Newline missing

Reply:
Why do we have starknet_in_rust here?

Reply:
[fix pyright config](https://github.com/software-mansion/protostar/pull/2059/commits/d0b32557a8b4f9e580f0849bc99799badc445afb)